dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/ValueTreeTableCell.java

```diff
@@ -17,14 +17,18 @@
 
 import javafx.scene.control.TreeTableCell;
 
-public class ValueTreeTableCell<T> extends TreeTableCell<T,Number> {
+public class ValueTreeTableCell<T> extends TreeTableCell<T, Number> {
     @Override
     protected void updateItem(final Number item, final boolean empty) {
         super.updateItem(item, empty);
         if (empty || item == null) {
             setText(null);
         } else {
-            setText(FileSizeFormat.getFileSizeType(item.longValue()));
+            if (item.longValue() == 0L) {
```

Comment:
I saw this same logic in another method.  Is this something that can be consolidated?

Reply:
Other method has been deleted.